src/update/balrog.cpp

```diff
@@ -25,10 +25,8 @@
 #  include "windows.h"
 #  include "platforms/windows/windowscommons.h"
 #  include "platforms/windows/golang-msvc-types.h"
```

Comment:
Do we still need this?

Reply:
Yes, we still need to define the C/golang types manually, since the C/go compiler doesn't produce an importable header file on the MSVC toolchain.